schema.json

```
@@ -1727,7 +1727,7 @@
         },
         "preferred-citation": {
             "$ref": "#/definitions/reference",
-            "description": "When citing the work, cite this reference instead."
+            "description": "The reference that should be cited when citing the work, instead of using the software or dataset metadata in this CITATION.cff file."
```

This is a bit confusing, because the metadata for the preferred-citation is also in "this CITATION.cff file".

---

I see, you're right. How about

```suggestion
            "description": "A reference to another work that should be cited instead of the software or dataset itself."
```

Better @jspaaks?

---

yessir